docs/A-Introduction/information.md

```diff
@@ -30,9 +34,12 @@ Since bits are too numerous to handle individually, modern computers transfer an
 
 The fundamental addressable unit of RAM is the byte.  One byte consists of 2 nibbles.  Each nibble consists of 4 bits.
 
-![](/img/bytenibbit.jpg)
+<div  align="left">
+<img src="../../static/img/bytenibbit.jpg" alt="A bytes tree image" />
```

Comment:
The `alt` text could be more descriptive. For instance: `Tree showing relative sizes of bit, nibble and byte`